Manual Testing/Ampeg Manual Test Project.txt

```diff
@@ -0,0 +1,68 @@
+Test Case #1 
+Create Account with Valid Information
+Entering valid information in create account page
+Pre conditions -Email, Username, First Name, Last Name, Password, Reenter Password, Country
+Traceability to Requirement - Create Account with Application
+Step 1 - Click Account Icon
+Step 2 - Click "Create Account" button
+Step 3 - Enter Valid Information
+Step 4 - Click "Create your account" button
+Expected Result - Account is successfully created and 
+Account Home Page is loaded.
+
+Test Case #2
+Create Account with Invalid Information
+Enter invalid username in create account page
+Pre conditions -Email, Username, First Name, Last Name, Password, Reenter Password, Country
```

Comment:
Same.
Pre-condition is that the user account already exists.

Reply:
Same thing, will change before merge